### ISSUE TITLE
Registration request for Acyrthosiphon pisum NCBI assembly

### ISSUE THREAD

**Issue by Scottyzy** [Bioconductor/GenomeInfoDb#36]
Hi,
I'd like to have the genome for Acyrthosiphon pisum registered for the purpose of forging a BSgenome package. The assembly is pea_aphid_22Mar2018_4r6ur  and below is the link to the NCBI page:

https://www.ncbi.nlm.nih.gov/assembly/GCF_005508785.1

Let me know if there's anything I can clear up or help with. Thank you very much.

**Comment by hpages** [comment]
Done in **GenomeInfoDb** 1.31.2 (see commit 513a854cb990dc6229f86af29f0fbc389bcb2a7f).

Note that whoever submitted the pea_aphid_22Mar2018_4r6ur assembly to NCBI messed it up:
- The name of the assembly is long and ugly.
- The reported assembly level should have been Scaffold, not Chromosome.
- The sequence names are terrible:
```
library(GenomeInfoDb)
Seqinfo(genome="pea_aphid_22Mar2018_4r6ur")
# Seqinfo object with 21920 sequences (1 circular) from pea_aphid_22Mar2018_4r6ur genome:
#   seqnames                    seqlengths isCircular                    genome
#   Scaffold_21773;HRSCAF=24826  132544852      FALSE pea_aphid_22Mar2018_4r6ur
#   Scaffold_20849;HRSCAF=22316  170740645      FALSE pea_aphid_22Mar2018_4r6ur
#   Scaffold_21967;HRSCAF=25451  119541763      FALSE pea_aphid_22Mar2018_4r6ur
#   Scaffold_21646;HRSCAF=24477   42333646      FALSE pea_aphid_22Mar2018_4r6ur
#   MT                               16971       TRUE pea_aphid_22Mar2018_4r6ur
#   ...                                ...        ...                       ...
#   Scaffold_21962;HRSCAF=25426       1664      FALSE pea_aphid_22Mar2018_4r6ur
#   Scaffold_21963;HRSCAF=25436       7008      FALSE pea_aphid_22Mar2018_4r6ur
#   Scaffold_21964;HRSCAF=25444      82198      FALSE pea_aphid_22Mar2018_4r6ur
#   Scaffold_21965;HRSCAF=25446      38071      FALSE pea_aphid_22Mar2018_4r6ur
#   Scaffold_21966;HRSCAF=25449     558143      FALSE pea_aphid_22Mar2018_4r6ur
```

The [Acyr_2.0 assembly](https://www.ncbi.nlm.nih.gov/assembly/GCF_000142985.2/) from the Baylor College of Medicine seems cleaner (but is also a few years older). Also registered in **GenomeInfoDb**:
```
Seqinfo(genome="Acyr_2.0")
# Seqinfo object with 23925 sequences (1 circular) from Acyr_2.0 genome:
#   seqnames      seqlengths isCircular   genome
#   MT                 16971       TRUE Acyr_2.0
#   Scaffold1        2622152      FALSE Acyr_2.0
#   Scaffold2        2384549      FALSE Acyr_2.0
#   Scaffold3        1975979      FALSE Acyr_2.0
#   Scaffold4        2478080      FALSE Acyr_2.0
#   ...                  ...        ...      ...
#   Scaffold23920        605      FALSE Acyr_2.0
#   Scaffold23921        430      FALSE Acyr_2.0
#   Scaffold23922       1480      FALSE Acyr_2.0
#   Scaffold23923        812      FALSE Acyr_2.0
#   Scaffold23924        683      FALSE Acyr_2.0
```
H.

**Comment by Scottyzy** [comment]
Hi just a quick question I was wondering if the genome will be available later on? as I type available.genomes() in R and I cant find the file

**Comment by Scottyzy** [comment]
Hi I was wondering how long before the assembly is available to use, as when i tried to run my seed files it said is not registered NCBI assembly. Thank you

**Comment by hpages** [comment]
Well, you need **GenomeInfoDb** 1.31.2. You can install it now (by installing it directly from GitHub) or wait that it becomes available in BioC 3.15 (this is BioC devel, it requires R 4.2), at which point you'll be able to install it via `BiocManager::install()`. This should take between 24h-48h.
